app/javascript/components/patient/Patient.js

```diff
@@ -68,665 +91,621 @@ class Patient extends React.Component {
-                      <div className="h5">(Edit)</div>
-                    </Button>
-                  )}
+                  <b>State/Local ID:</b> <span>{this.props.details.user_defined_id_statelocal || '--'}</span>
+                </div>
+                <div>
+                  <b>CDC ID:</b> <span>{this.props.details.user_defined_id_cdc || '--'}</span>
+                </div>
+                <div>
+                  <b>NNDSS ID:</b> <span>{this.props.details.user_defined_id_nndss || '--'}</span>
                 </div>
-                <div className="clearfix"></div>
               </Col>
-            </Row>
-            <Row>
-              <Col className="text-truncate">
-                <Row>
-                  <Col>
-                    <b>Phone:</b> <span>{this.props.details.primary_telephone ? `${this.formatPhoneNumber(this.props.details.primary_telephone)}` : '--'}</span>
-                    {this.props.details.blocked_sms && (
-                      <Form.Label className="tooltip-whitespace nav-input-label font-weight-bold">
-                        &nbsp;SMS Blocked <InfoTooltip tooltipTextKey="blockedSMS" location="top"></InfoTooltip>
-                      </Form.Label>
-                    )}
-                  </Col>
-                </Row>
-                <Row>
-                  <Col>
-                    <b>Preferred Contact Time:</b> <span>{this.props.details.preferred_contact_time || '--'}</span>
-                  </Col>
-                </Row>
-                <Row>
-                  <Col>
-                    <b>Type:</b> <span>{this.props.details.primary_telephone_type || '--'}</span>
-                  </Col>
-                </Row>
-                <Row>
-                  <Col>
-                    <b>Email:</b> <span>{this.props.details.email || '--'}</span>
-                  </Col>
-                </Row>
-                <Row>
-                  <Col>
-                    <b>Preferred Reporting Method:</b>{' '}
-                    {(!this.props.details.blocked_sms || !this.props.details.preferred_contact_method?.includes('SMS')) && (
-                      <span className="font-weight-light">{this.props.details.preferred_contact_method || '--'}</span>
-                    )}
-                    {this.props.details.blocked_sms && this.props.details.preferred_contact_method?.includes('SMS') && (
-                      <span className="font-weight-bold text-danger">
-                        {this.props.details.preferred_contact_method || '--'}
-                        <Form.Label className="tooltip-whitespace">
-                          <InfoTooltip tooltipTextKey="blockedSMSContactMethod" location="top"></InfoTooltip>
-                        </Form.Label>
-                      </span>
-                    )}
-                  </Col>
-                </Row>
+              <Col sm={14} className="item-group">
```

Comment:
I'd default to @mrnosal's opinion but I don't think so because most of the time everything is one line in height and that is even with 6 on each side.

Reply:
I'd suggest leaving as is. Users are familiar with this grouping already, and most times there is only one line of height for the values.